src/views/all_news.hbs

```diff
@@ -1,17 +1,17 @@
-<div >
-  <h1>Tech & Startup News</h1>
-  <div>
-    <div>
-      <a href="/news/{{_id}}"><img src="/assets/fac.png"></a>
-      <a href="/news/{{_id}}">{{this.SingleNews.newsTitle}}</a>
-      <p>{{this.news.subhead}}</p>
-      <a href="/news/{{_id}}">Read More</a>
+<div class="w-60 center db">
+  <h1 class="fl tracked mt4 mid-gray">Tech & Startup News</h1>
```

Comment:
i did use text-granite-gray for the paragraphs but the title needs darker text thats why i used a tachyons color mid-gray